dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilderUtility.h

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+#include "pch.h"
+#include "winrt/Microsoft.Windows.AppNotifications.Builder.h"
+
+inline const size_t c_maxAppNotificationPayload{ 5120 };
+inline const uint8_t c_maxTextElements{ 3 };
+inline const uint8_t c_maxButtonElements{ 5 };
+
+namespace AppNotificationBuilder
+{
+    using namespace winrt::Microsoft::Windows::AppNotifications::Builder;
+}
+
+inline std::wstring GetWinSoundEventString(AppNotificationBuilder::AppNotificationSoundEvent soundEvent)
```

Comment:
NIT: Returning these string constants as string objects means malloc+free hits. PCWSTR or maybe `std::wstring_view` is likely more efficient (depends on callers but usually the case)

Reply:
Addressed in https://github.com/microsoft/WindowsAppSDK/pull/2801